azure-iot-device/tests/common/pipeline/test_pipeline_stages_mqtt.py

```diff
@@ -248,9 +253,11 @@ def test_sets_pending_operation(self, stage, create_transport, op_connect):
     @pytest.mark.parametrize(
         "pending_connection_op",
         [
-            pytest.param(pipeline_ops_base.ConnectOperation(), id="Pending ConnectOperation"),
-            pytest.param(pipeline_ops_base.ReconnectOperation(), id="Pending ReconnectOperation"),
-            pytest.param(pipeline_ops_base.DisconnectOperation(), id="Pending DisconnectOperation"),
+            pytest.param(pipeline_ops_base.ConnectOperation(1), id="Pending ConnectOperation"),
+            pytest.param(pipeline_ops_base.ReconnectOperation(1), id="Pending ReconnectOperation"),
+            pytest.param(
+                pipeline_ops_base.DisconnectOperation(1), id="Pending DisconnectOperation"
```

Comment:
>1 [](start = 54, length = 1)

This is an unpleasant side effect of the callback being required - and we can't use fixtures because of the parametrization (WHY PYTEST WHY?).

I'd probably prefer use of None to use of integer 1, but eh, neither is great.